app/models/conversion_host.rb

```diff
@@ -251,10 +251,10 @@ def find_credentials(msg = nil)
       resource.authentication_type('default')
 
     unless authentication
-      msg = "Credentials not found for conversion host #{name} or resource #{resource.name}"
-      msg << " #{msg}" if msg
-      _log.error(msg)
-      raise MiqException::Error, msg
+      error_msg = "Credentials not found for conversion host #{name} or resource #{resource.name}"
+      error_msg << " #{msg}" if msg
+      _log.error(error_msg)
+      raise MiqException::Error, error_msg
```

Comment:
@djberg96 is this related to the credential issue?

Reply:
Oh I see `msg` is being passed in and overwritten...I don't see any callers passing in a message so why not just drop the parameter?

Reply:
As per our gitter discussion, I think I was trying to preserve the original code a little too faithfully. It probably isn't needed, but I can refactor that out in a separate PR.

Reply:
I'd rather do that in this pr and change it for backport if you want, or put the refactoring PR in now so we don't forget it.

E.g. we never followed up and used the new MiqTask interface that you were going to follow up with here https://github.com/ManageIQ/manageiq/blob/master/app/models/conversion_host/configurations.rb

Reply:
Not yet, but will. Waiting for a breather.